<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeRenamer</class>
 <widget class="QWidget" name="NodeRenamer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_prefix" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_prefix">
        <property name="text">
         <string>Prefix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_prefix"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_name" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_name"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_suffix" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_suffix">
        <property name="text">
         <string>Suffix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_suffix"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_buttons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="button_prefix">
        <property name="text">
         <string>Prefix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_name">
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_suffix">
        <property name="text">
         <string>Suffix</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list_preview"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
